### PR TITLE
A few typos and adjusted wording slightly to sound better.

### PR DESCRIPTION
**Pull request by wayneeseguin**
Corrected a few typos and adjusted wording slightly to sound better.

### CODE REVIEW

**Comment by wayneeseguin** [comment]
Quick question about one of the code examples in the functions chapter.

I had thought that I recalled the common go idiom is that functions returned {value}, {ok} and this example is swapped where ok is returned first. Am I recalling incorrectly?

**Comment by initpy** [comment]
Now this is a _HUGE_ contribution! Thank you so much, Wayne! I really appreciate your help!

File: functions.rst [hunk 1]
```diff
@@ -413,32 +415,33 @@ Functions signatures
 ====================
 Like with people, names don't matter that much. Do they? Well, yes, let's face
 it, they do matter, but what matters most is what they do, what they need, what
-they produce.
+they produce (Yeah you! Get back to work!).
```
Comment (initpy) [inline functions.rst:418]: This actually made me smile! Thanks for the initiative :)

**Comment by initpy** [comment]
Oh, yes, I think you're totally right about the "comma-ok" idiom. I should fix that, thanks for pointing it out to me! And again, sorry for my poor English. I bet I'm making your -and many others- eyes bleed :)

**Comment by wayneeseguin** [comment]
:) Just a bit ;)  But that's fine, I love the book you are writing so I am happy to help with your Engrish.

**Comment by initpy** [comment]
You, definitely, are my hero today! I'm really grateful :)

**Pull request merged**